css/portal-dashboard/header.less

```diff
@@ -36,11 +36,26 @@
     background-color: @cc-orange;
   }
 
-  .assignment {
+  .headerCenter {
     display: flex;
+    flex-direction: row;
     flex: 1;
     justify-content: center;
-    margin: 0 10px 0 10px;
+    margin: 15px 0px 15px 0px;
+    align-items: center;
+    width: 400px;
+    height: 100%;
+    font-size: 16px;
+
+    .assignmentTitle{
+      margin-right: 10px;
+      padding-bottom:3px;
+    }
+    .chooseAssignment{
+      width: 300px;
+      height: 30px;
```

Comment:
I'm guessing similarly we won't need this width and height if we restructure the custom select.